src/includer/parsers.ts

```diff
@@ -110,6 +110,15 @@ function tagsFromSpec(spec: OpenAPISpec): Map<string, Tag> {
         }
     }
 
+    parsed.forEach((tag: Tag) => {
+        if (tag['x-navtitle']) {
```

Comment:
I assume this is supposed to make prior `x-navtitle` resolution logic (defined in this file on L94) obsolete, would you say that is correct?

Reply:
Yes. It overrides `x-navtitle` from endpoints. It was always looks very unnatural and error-prone to me. I mean, what if different endpoints uses different `x-navtitle` for the same tag?

But I'm trying to keep backward compatibility.